README.rst

```diff
@@ -479,10 +479,16 @@ Full List of Settings
   http://readthedocs.org/docs/boto/en/latest/ref/ses.html#boto.ses.regions
   http://docs.aws.amazon.com/general/latest/gr/rande.html
 
-``AWS_SES_RETURN_PATH``
-  Instruct Amazon SES to forward bounced emails and complaints to this email.
+``AWS_SES_FROM_EMAIL``
+  Optional. The email addres to be used as the "From" address for the email. The address that you specify has to be verified.  
   For more information please refer to http://aws.amazon.com/ses/faqs/#38
 
+``AWS_SES_RETURN_PATH``
+  Optional. Use `AWS_SES_RETURN_PATH` to receive complaint notifications
+  It only works in the v2 client.
```

Comment:
Should this value be used only if `USE_SES_V2` is `True`?

Reply:
Yes, it will only affect emails sent using `USE_SES_V2`.

The V1 WONT'T use this attribute or it's value!

Reply:
```suggestion
  You must use the v2 client by setting `USE_SES_V2=True` for this setting to work, otherwise it is ignored.
```